### PR TITLE
Fix blog banner layout

### LINKED ISSUE #2297: [Bug]: New Blog Notification is too large
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have searched the [issue tracker](https://github.com/FreeTubeApp/FreeTube/issues) for a bug report that matches the one I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.

### Describe the bug

1. Install a fresh version of FreeTube Nightly (i installed v0.16.0-nightly-1708 beta) or a new blog could be simulated for testing purposes (settings would have to be reset otherwise)
2. Open the app
3. See a new blog notification
4.  see bug
5. 
image

### Expected Behavior

New blog notification should be smaller

### Issue Labels

visual bug

### FreeTube Version

v0.16.0-nightly-1708 Beta

### Operating System Version

Windows 10

### Installation Method (If applicable)

.exe

### Last Known Working FreeTube Version (If Any)

_No response_

### Primary API used

_No response_

### Additional Information

_No response_

### Nightly Build

- [X] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

### PR DESCRIPTION
**Pull request by vallode**
---
Title
---

**Pull Request Type**
Please select what type of pull request this is:
- [x] Bugfix

**Related issue**
Closes #2297

**Description**
Fixes a layout error with the blog notification banner

**Screenshots (if appropriate)**
Before:
image

After:
image

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] 10
 - OS Version: [e.g. 22] Debian
 - FreeTube version: [e.g. 0.8] upstream/development

**Additional context**
Cheers chonky for pointing it out!

### CODE REVIEW

File: src/renderer/App.vue [hunk 1]
```diff
@@ -9,10 +9,8 @@
   >
     <top-nav ref="topNav" />
     <side-nav ref="sideNav" />
-    <ft-flex-box
-      v-if="showUpdatesBanner || showBlogBanner"
```
[the remark this answers is not in the document]
Comment (vallode) [reply src/renderer/App.vue:13]: I nested the routerview within this and the v-ifs were already on the individual elements anyway, this statement is unnecessary now.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
could u also create a PR in the RC branch?

**Review by PikachuEXE — APPROVED** [review]
Local test result:

Before fix
image

After fix
image

**Comment by absidue** [comment]
This PR unfortunately doesn't fix the issue entirely on my end.

**New version banner**
Testing:
1. change version in package.json to `0.15.0`
2. `yarn run dev`

Result:
new_version_banner

**New blog banner**
Testing:
1. `yarn run dev`
2. In the devtools: Application -> Local Storage -> `http://localhost:9080`
3. Delete `lastAppWasRunning` key
4. Close FreeTube
5. `yarn run dev`

Result:
new_blog_banner

**Review by absidue — CHANGES_REQUESTED** [review]
Looks like the banner is greedy and tries to take up as much space as possible.

Loading screen while searching:
loading_screen

Search results:
search_results

**Comment by PikachuEXE** [comment]
@vallode 
I tried to fix it and here is my maybe WIP result
https://github.com/FreeTubeApp/FreeTube/compare/development...PikachuEXE:FreeTube:fix-blog-notification-pika

https://user-images.githubusercontent.com/1018543/177257127-0eb4a1f9-dcd5-486d-93fd-3336b3ff6c76.mov

**Comment by PikachuEXE** [comment]
You can experiment with more properties with
https://the-echoplex.net/flexyboxes/?fixed-height=on&display=flex&flex-direction=column&flex-wrap=nowrap&justify-content=flex-start&align-items=center&align-content=stretch&order%5B%5D=0&flex-grow%5B%5D=0&flex-shrink%5B%5D=1&flex-basis%5B%5D=auto&align-self%5B%5D=auto&order%5B%5D=0&flex-grow%5B%5D=0&flex-shrink%5B%5D=1&flex-basis%5B%5D=auto&align-self%5B%5D=auto&order%5B%5D=0&flex-grow%5B%5D=1&flex-shrink%5B%5D=1&flex-basis%5B%5D=auto&align-self%5B%5D=auto

**Comment by vallode** [comment]
Thanks everyone for the reminders, been going through some personal stuff. Fixed it for now, should I open a new PR for the release candidate too?

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
>  should I open a new PR for the release candidate too?

@vallode yes please! :)

**Review by absidue — APPROVED** [review]
Thank you, looks good now

**Review by GilgusMaximus — APPROVED** [review]
Tested locally, works well. No problems at different sizes or with different amount of videos

**Pull request merged**